### ISSUE TITLE
Can we have improved pie chart display for checks running 24 hours in preview mode

### ISSUE THREAD

**Issue by aliasgerkw** [checkly/public-roadmap#134]
💡 For general support requests and bug reports, please go to [checklyhq.com/support](https://checklyhq.com/support)

**Is your feature request related to a problem? Please describe.**
Checks which are scheduled for 24 hours, can we show the graph with monthly x axis in this preview?

**Comment by aliasgerkw** [comment]
<img width="713" alt="Screenshot 2021-06-24 at 11 02 03 AM" src="https://user-images.githubusercontent.com/51911147/123207822-9d7be180-d4db-11eb-8bbb-ec9537697109.png">

**Comment by tnolet** [comment]
@aliasgerkw yeah, this is very sub optimal for sparsely run checks. We will have a look on how to improve this soon.
